earthpy/spatial.py

```diff
@@ -121,37 +121,31 @@ def normalized_diff(b1, b2):
 
 
 # TODO: include a no data value here if provided
+def stack(band_paths, out_path=""):
 
-
-def stack_raster_tifs(band_paths, out_path, arr_out=True):
     """Take a list of raster paths and turn into an output raster stack
-    numpy array. Note that this function depends upon the stack() function.
-
-    TODO: Instead of returning a file path when arr_out=False, consider
-        returning None since the out_path is already
-        an input given by the user. This will make the output type consistent.
+    numpy array. Note that this function depends upon the stack_bands() function.
 
     Parameters
     ----------
     band_paths : list of file paths
         A list with paths to the bands you wish to stack. Bands
         will be stacked in the order given in this list.
-    out_path : string
+    out_path : string (optional)
         A path with a file name for the output stacked raster
          tif file.
-    arr_out : boolean
+    write_raster : boolean (optional)
```

Comment:
@joemcglinchy if `write_raster` is no longer in thie function can we please remove it from the docstring? then travis is failing i think because the tests and such call it too.

Reply:
yep done. write_raster is removed from test file too.